alacritty_terminal/src/term/mod.rs

```diff
@@ -920,7 +920,8 @@ impl<T> Term<T> {
 
         let history_size = config.scrolling.history() as usize;
         let grid = Grid::new(num_lines, num_cols, history_size, Cell::default());
-        let alt = Grid::new(num_lines, num_cols, 0 /* scroll history */, Cell::default());
+        let inactive_grid =
+            Grid::new(num_lines, num_cols, 0 /* scroll history */, Cell::default());
```

Comment:
```suggestion
        let alt_grid = Grid::new(num_lines, num_cols, 0, Cell::default());
```

Never was a big fan of this comment and especially with this linebreak it's probably better to get rid of it.

But I think I'd like to keep this instance as `alt_grid`, since at this point we need for certain that this is actually the alt grid and thus the `0` history size makes a lot of sense.

Just using `inactive_grid: alt_grid` in the `Term` creation should be fine.